atlasdb-container-test-utils/src/main/java/com/palantir/atlasdb/containers/CassandraContainer.java

```diff
@@ -72,14 +72,15 @@
     public String getDockerComposeFile() {
         return "/docker-compose-cassandra.yml";
     }
-
+    // TODO(gmaretic): for now kept synchronous initialization -- should revisit
     @Override
     public SuccessOrFailure isReady(DockerComposeRule rule) {
         return SuccessOrFailure.onResultOf(() -> {
-            CassandraKeyValueService.create(
+            CassandraKeyValueServiceImpl cassandraKeyValueService = CassandraKeyValueServiceImpl.create(
                     CassandraKeyValueServiceConfigManager.createSimpleManager(KVS_CONFIG),
-                    LEADER_CONFIG);
-            return true;
+                    LEADER_CONFIG,
+                    false);
```

Comment:
If this is sync, the readiness check should be good if the creation succeeds right? `cassandraKeyValueService.isInitialized()` need not be checked here.

Reply:
This is fixed in #2300. Since it's just tests, we probably shouldn't fix it here.